tests/admin/test_controller.py

```diff
@@ -33,6 +33,7 @@ class AdminControllerTest(ControllerTest):
     def setup(self):
         with temp_config() as config:
             config[Configuration.INCLUDE_ADMIN_INTERFACE] = True
+            config[Configuration.SECRET_KEY] = "a secret"
```

Comment:
These fix test failures on a minimally-configured system by making sure a basic config is in place. There are still two failing tests in tests/admin I wasn't able to fix. The failures didn't prevent earlier circulation branches from being merged, making me think that these tests aren't being run automatically.